src/libraries/System.Linq.Expressions/src/System/Dynamic/DynamicObject.cs

```diff
@@ -145,7 +146,7 @@ public virtual bool TryInvoke(InvokeBinder binder, object[] args, out object res
         /// <param name="result">The result of the operation.</param>
         /// <returns>true if the operation is complete, false if the call site should determine behavior.</returns>
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1007:UseGenericsWhereAppropriate")]
-        public virtual bool TryBinaryOperation(BinaryOperationBinder binder, object arg, out object result)
+        public virtual bool TryBinaryOperation(BinaryOperationBinder binder, object arg, out object? result)
```

Comment:
>object arg [](start = 77, length = 10)

Should this be `object? arg` instead?

Reply:
In the only implementation I found it requires non null handler for `InPlaceAdd(object handler)` and `InPlaceSubtract(object handler)` methods: http://index/?leftProject=System.Core&leftSymbol=kyj8cmwk9atr&rightProject=System.Dynamic&file=System%5CDynamic%5CBoundDispEvent.cs&line=44